### PR TITLE
Switching to Keras 1.0; Some PEP8-friendly beautifications; additional hidden layer

### PR DESCRIPTION
**Pull request by gb96**
Thanks to @Wildream for these changes
Note the use of num_recurrent_units parameter which adds additional hidden layers to the model.

### CODE REVIEW

File: convert_directory.py [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-clip_len = 10 		#length of clips for training. Defined in seconds
-block_size = freq / 4 #block sizes used for training - this defines the size of our input state
-max_seq_len = int(round((freq * clip_len) / block_size)) #Used later for zero-padding song sequences
-#Step 1 - convert MP3s to WAVs
-new_directory = convert_folder_to_wav(input_directory, freq)
-#Step 2 - convert WAVs to frequency domain with mean 0 and standard deviation of 1
-convert_wav_files_to_nptensor(new_directory, block_size, max_seq_len, output_filename)
\ No newline at end of file
+freq = config['sampling_frequency']  # sample frequency in Hz
+clip_len = 10  # length of clips for training. Defined in seconds
```
Comment (gb96) [inline convert_directory.py:10]: should define this in nn_config

File: data_utils/parse_files.py [hunk 2]
```diff
@@ -2,196 +2,225 @@
...
+    y_data[:][:] /= std_x  # Variance 1
+
+    np.save(out_file + '_mean', mean_x)
+    np.save(out_file + '_var', std_x)
+    np.save(out_file + '_x', x_data)
+    np.save(out_file + '_y', y_data)
+    print 'Done!'
+
+
+def nptensor_to_wav_files(tensor, indices, filename, useTimeDomain=False):
```
Comment (gb96) [inline data_utils/parse_files.py:184]: note renamed function

File: train.py [hunk 3]
```diff
@@ -11,41 +11,53 @@
...
 
-#Creates a lstm network
-model = network_utils.create_lstm_network(num_frequency_dimensions=freq_space_dims, num_hidden_dimensions=hidden_dims)
-#You could also substitute this with a RNN or GRU
-#model = network_utils.create_gru_network()
+# Creates a lstm network
+model = network_utils.create_lstm_network(
+    num_frequency_dimensions=freq_space_dims,
+    num_hidden_dimensions=hidden_dims,
+    num_recurrent_units=2
```
Comment (gb96) [inline train.py:30]: Note additional hidden layers.  Perhaps also need to add some dropout between these layers?
